### ISSUE TITLE
ThirdParty HSL seems broken

### ISSUE THREAD

**Issue by eslickj** [coin-or/coinbrew#34]
The coinbrew script is a little hard to follow, but I have an issue with it getting the Thirdparty-HSL build tools.  I think the issue is that the script checks if you have the HSL source code in place to decide if it should get the build tools, but with the source code in place it can't clone the HSL build tools repo there.

I worked around it here: https://github.com/IDAES/coinbrew/tree/idaes-stable, so I fetch first then copy the code in between fetch and build.  Probably not the best solution, but it will do until this gets fixed.

**Comment by tkralphs** [comment]
Yeah, sorry, looks like I botched the attempt at fixing a different issue regarding HSL (#28). The problem is that I was using the same function to determine whether to fetch and whether to build. The same logic works for all cases, but HSL. Unfortunately, though, you do need to fetch, then add the source code, then build. There's really no way around that, since we don't have any way of automatically fetching the source like we do for the other projects.

**Comment by eslickj** [comment]
Thanks.  It's working.
